### ISSUE TITLE
App crash against Kubernetes 1.20

### ISSUE THREAD

**Issue by cvandesande** [lensapp/lens#1711]
**Describe the bug**
Lens displays App crash after connecting to a Kubeadm based 1.20.0 cluster for a minute or so.

**To Reproduce**
1. Upgrade a Kubernetes cluster from 1.19.4 to 1.20.0
2. Open Lens.
3. Do nothing, or click a few things, or wait for about 1 min.
4. See App Crash

**Environment (please complete the following information):**
- Lens Version: 4.0.0
- OS: Debian 10.7 Buster
- Installation method AppImage

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
Error stack:

App crash at http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/cluster
 TypeError: Cannot read property 'kind' of undefined
    at r.value (http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:3776353)
    at http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:8680755
    at Array.map (<anonymous>)
    at http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:8680723
    at Array.map (<anonymous>)
    at r.value (http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:8680667)
    at r.value (http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:8686015)
    at ve (http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:18448)
    at http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:70625
    at ce (http://02bceedc-95b4-4c44-89dd-435741bb4e51.localhost:34031/build/Lens.js:2:16554)
```
**Kubeconfig:**
Hasn't changed since before the app crashes started
**Additional context**
Just upgraded the "bare metal" cluster with kubeadm to 1.20.0.  Cluster came up healthy, opened Lens 4.0.0 everything seemed fine, all things look normal. After about 60 seconds, App crash

**Comment by cvandesande** [comment]
Lens just updated to 4.0.1 (Nice work with the AppImage updates on Linux).
Still crashes anywhere between 10-60 seconds of use.
ctrl-r (reload) is a good workaround.

**Comment by Nascire** [comment]
Can confirm for Windows, Lens 4.0.1

**Comment by J3m5** [comment]
Same here

**Comment by fzalila** [comment]
same issue here.

**Comment by domrod** [comment]
Same issue using Lens 4.0.2 (on Debian buster).

I confirm that ctrl-r (reload) is a good workaround but crashes come later anyway. 
On the other hand, when clicking on a cluster node, no additional information folder opens anymore also.

**Comment by tece81** [comment]
> same issue here.

+1 Lens 4.0.3, k8s 1.20

**Comment by SergeyLadutko** [comment]
same issue here.

+1 Lens 4.0.3, k8s 1.20

**Comment by jakolehm** [comment]
Fixed in #1804
